src/pages/Wallet/WalletHomePage.tsx

```diff
@@ -357,7 +355,7 @@ const CompactWalletAmountBox = styled(motion.div)`
   }
 `
 
-const WalletAmount = styled(motion.div)`
+const WalletAmount = styled(motion(Amount))`
```

Comment:
The console error that I noticed is associated with this line.

Reply:
@mvaivre dont' ask me why, but this fixes the error:

```diff
diff --git a/src/components/Amount.tsx b/src/components/Amount.tsx
index 5cce0ce..7c92744 100644
--- a/src/components/Amount.tsx
+++ b/src/components/Amount.tsx
@@ -16,6 +16,8 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
+import { forwardRef } from 'react'
+
 import { useGlobalContext } from '../contexts/global'
 import { abbreviateAmount } from '../utils/numbers'
 
@@ -24,7 +26,8 @@ interface AmountProps {
   className?: string
 }
 
-const Amount = ({ value, className }: AmountProps) => {
+// eslint-disable-next-line react/display-name
+const Amount = forwardRef(({ className, value }: AmountProps, ref) => {
   const {
     settings: {
       general: { discreetMode }
@@ -36,6 +39,6 @@ const Amount = ({ value, className }: AmountProps) => {
       discreetMode ? '•••' : value !== undefined ? abbreviateAmount(value) : '-'
     } ℵ`}</span>
   )
-}
+})
 
 export default Amount

```

Reply:
Well, I've removed the motion wrapper which I believe is not needed, so we should be good to go!
I'll appreciate if you could review a last time, thanks :)